### PR TITLE
Update performance docs

### PR DESCRIPTION
**Pull request by jrcrespoh**
- [x] Update README.md
- [x] Add OUTLINE.md

### CODE REVIEW

File: performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+### updateStatusAsync
+```swift
+@MainActor @available(iOS 15, tvOS 15,*)
+func updateStatusAsync(to newStatus: ProcessStatus, updateUploadStatus: Bool = false)
+```
+updates the Process status asynchronously, possibly updating the status of the upload task
+
+### updateImageAsync
+```swift
+@MainActor @available(iOS 15, tvOS 15, *) func updateImageAsync(to newImage: UIImage?)
```
Comment (ncooke3) [inline performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md:141]: I think breaking these code snippets into two lines (one for function attributes, one for function signature) like done above would make them easier to read.

File: performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+tracks the different process tasks
+
+### ProcessStatus
+```swift
+enum ProcessStatus: Equatable {
+  case idle
+  case running(ProcessTask)
+  case failure(ProcessTask)
+  case success(ProcessTask)
+```
```
Comment (ncooke3) [inline performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md:255]: nit: missing closing `}`

File: performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+    - [body](#body-5)
+
+## PerformanceExampleApp
+```swift
+@main
+struct PerformanceExampleApp: App
+```
+main point of entry into app
+
+### init
```
Comment (ncooke3) [inline performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md:60]: optional: for these headers, you could make them inline code blocks
> ### `init`
> ```swift
> init()
> ```
> configures the FirebaseApp

**Review by ncooke3 — APPROVED** [review]

File: performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md (hunk 2 above)
Comment (jrcrespoh) [reply performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md:255]: That was intentional in that I wanted to imply that there was more to the enum (see `text` and `view` right after), but I do agree it is unclear. I'll add the closing `}` with some ellipses in-between.

Comment (jrcrespoh) [reply performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md:255]: Done ✅

File: performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md (hunk 3 above)
Comment (jrcrespoh) [reply performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md:60]: Done ✅

**Review by paulb777 — APPROVED** [review]

**Pull request merged**